plugins/ca-kafka-local/plugin.json

```diff
@@ -5,6 +5,9 @@
   "readme": "README.md",
   "env": {
     "KAFKA_BROKERS": "localhost:14231",
```

Comment:
For ease of use, do we want an environment variable for `localhost:14233` (KAFKA_BROKERS_SASL maybe)?

Reply:
I was thinking about changing the `KAFKA_BROKERS` to SASL port, but that will break exisiting usage. `KAFKA_BROKERS_SASL` is better.